src/components/cylc/table/Table.vue

```diff
@@ -240,40 +227,59 @@ export default {
         mdiChevronDown,
         mdiArrowDown
       },
-      sortBy: ['Cycle Point'],
+      sortBy: ['node.cyclePoint'],
       sortDesc: [localStorage.cyclePointsOrderDesc ? JSON.parse(localStorage.cyclePointsOrderDesc) : true],
       expanded: [],
       headers: [
         {
-          text: 'Task'
+          text: 'Task',
+          value: 'node.name',
+          sort: DEFAULT_COMPARATOR
```

Comment:
In order to get Vuetify's built-in sorting to work, it seems the `value` field should be the prop on the component that this column displays (and using the dot-chaining notation for child properties seems to work)

Reply:
I didn't personally notice this, but it makes sense. Good catch.